tests/test_main.py

```diff
@@ -20,4 +21,7 @@
 def test_exception_handling(monkeypatch):
     replaced_dispatch = pretend.raiser(exceptions.InvalidConfiguration("foo"))
     monkeypatch.setattr(cli, "dispatch", replaced_dispatch)
-    assert dunder_main.main() == "InvalidConfiguration: foo"
+    assert (
```

Comment:
Is there a way to assert this doesn't format things with color in the event that we're not talking to a `tty`

Reply:
There should be a way to do that; I'll make sure to loop back around to this once I'm done with the GitHub Actions CI pipeline and I start writing more tests specifically for colorama.

Reply:
I think this is probably colorama's responsibility to test, not ours ([and they are already handling it](https://github.com/tartley/colorama/blob/bfa8c22e8f9b51ad3428c165087540cea5d44a1f/colorama/ansitowin32.py#L64-L72) and [testing it](https://github.com/tartley/colorama/blob/master/colorama/tests/isatty_test.py)).